Package.swift

```diff
@@ -2,9 +2,19 @@ import PackageDescription
 
 let package = Package(
     name: "ReactiveSwift",
-    dependencies: [
-        .Package(url: "https://github.com/antitypical/Result.git", majorVersion: 3, minor: 0)
-    ],
+    dependencies: {
+        #if os(macOS)
+            return [
+                .Package(url: "https://github.com/antitypical/Result.git", majorVersion: 3, minor: 0),
+            ]
+        #else
+            return [
+                .Package(url: "https://github.com/antitypical/Result.git", majorVersion: 3, minor: 0),
+                .Package(url: "https://github.com/Quick/Nimble", majorVersion: 5, minor: 0),
+                .Package(url: "https://github.com/Quick/Quick", majorVersion: 0, minor: 10),
+            ]
+        #endif
```

Comment:
Unfortunately conditional compilation like that does not work:

```
ReactiveCocoa/ReactiveSwift/Package.swift:8:9: error: expected expression in container literal
        #if !os(macOS)
        ^
ReactiveCocoa/ReactiveSwift/Package.swift:7:99: error: expected ',' separator
            .Package(url: "https://github.com/antitypical/Result.git", majorVersion: 3, minor: 0),
                                                                                                  ^
                                                                                                  ,
ReactiveCocoa/ReactiveSwift/Package.swift:7:99: error: expected ']' in container literal expression
            .Package(url: "https://github.com/antitypical/Result.git", majorVersion: 3, minor: 0),
                                                                                                  ^
ReactiveCocoa/ReactiveSwift/Package.swift:6:16: note: to match this opening '['
        return [
               ^
ReactiveCocoa/ReactiveSwift/Package.swift:9:14: error: unexpected platform condition (expected 'os', 'arch', or 'swift')
            .Package(url: "https://github.com/Quick/Nimble", majorVersion: 5, minor: 0),
             ^
ReactiveCocoa/ReactiveSwift/Package.swift:9:88: error: extra tokens following conditional compilation directive
            .Package(url: "https://github.com/Quick/Nimble", majorVersion: 5, minor: 0),
                                                                                       ^
ReactiveCocoa/ReactiveSwift/Package.swift:9:88: error: expected expression
            .Package(url: "https://github.com/Quick/Nimble", majorVersion: 5, minor: 0),
                                                                                       ^
ReactiveCocoa/ReactiveSwift/Package.swift:12:9: error: expected expression
        ]
        ^
ReactiveCocoa/ReactiveSwift/Package.swift:13:6: error: expected ',' separator
    }(),
     ^
     ,
ReactiveCocoa/ReactiveSwift/Package.swift:13:6: error: unnamed argument #3 must precede argument 'dependencies'
    }(),
~~~~~^
()
Can't parse Package.swift manifest file because it contains invalid format. Fix Package.swift file format and try again.
error: invalidManifestFormat
```